<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>365</width>
    <height>545</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>365</width>
    <height>545</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	
	background-color:rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="numeroQuatro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroCinco">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroSeis">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subtracao">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adicao">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroUm">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroTres">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroDois">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="igual">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>=, Return, Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroZero">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 35px;</string>
    </property>
    <property name="text">
     <string>0        </string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="virgula">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
    <property name="shortcut">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplicacao">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroSete">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroNove">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numeroOito">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: rgb(84, 84, 84);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divisao">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="limparTudo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color:rgb(148, 148, 148);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPorcentagem">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color:rgb(148, 148, 148);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="shortcut">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maisMenos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">margin: 5px;
    background-color:rgb(148, 148, 148);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 40px;
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultado">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
